charts/preview-app/values.yaml

```diff
@@ -4,14 +4,14 @@ imageTag: 'staging'
 commitHash: ''
 projects:
   - name: ''
-
-otherContainers:
-
-      
+otherContainers: []
+service: {}
 volumes: false
-volumeMounts:
+volumeMounts: {}
 useJobs: false
-jobs:
+jobs: {}
+ingress:
+  enabled: false
```

Comment:
Mohli bychom tu dát true? Když teď nějaký projekt upgradne na tuhle verzi tak mu vypadne ingress pokud ho explicitně nezapne.

Reply:
jasně, upravím to